src/v0/__tests__/analytics.spec.ts

```diff
@@ -0,0 +1,129 @@
+import fetchMock from 'fetch-mock';
+import createAnalytics, { AnalyticsEvent, ErrorEvent } from '../analytics';
+
+const local = 'begin:http://analytics.arigato.tools';
+const stage = 'begin:https://analytics.stage.manifold.co';
+const prod = 'begin:https://analytics.manifold.co';
+
+const metric: AnalyticsEvent = {
+  type: 'metric',
+  name: 'rtt_graphql',
+  properties: {
+    duration: 123,
+  },
+};
+const error: ErrorEvent = {
+  type: 'error',
+  name: 'mui-pricing-matrix_error',
+  properties: {
+    code: 'code',
+    message: 'message',
+  },
+};
+const track: AnalyticsEvent = {
+  type: 'component-analytics',
+  name: 'click',
+  properties: {
+    planId: '1234',
+  },
+};
+
+describe('analytics', () => {
```

Comment:
I copied all of these from the plan-table repo and modified them slightly to work with how we're doing it here. @TerrySmithDC if you look through it you'll probably notice the biggest difference: we specify the element, component version, and env just once in `createAnalytics` and then we don't worry about them again.